cht-config/app_settings.json

```diff
@@ -0,0 +1,590 @@
+      ]
+    }
+  },
+  "create_user_for_contacts": {
+    "replace_forms": [
+      "replace_user"
+    ]
+  },
+  "transitions": {
+    "accept_patient_reports": false,
+    "conditional_alerts": false,
+    "default_responses": false,
+    "update_sent_by": false,
+    "registration": false,
+    "update_clinics": false,
+    "update_notifications": false,
+    "update_scheduled_reports": false,
+    "generate_patient_id_on_people": true,
+    "death_reporting": true,
+    "create_user_for_contacts": false,
+    "mark_for_outbound": true
+  },
+  "outbound": {
+    "FHIR_patient": {
+      "relevant_to": "doc.type === 'person' && doc.role == 'patient'",
+      "destination": {
+        "base_url": "http://localhost:5001",
+        "path": "/mediator/patient",
+        "auth": {
+          "type": "basic",
+          "username": "interop-client",
+          "password_key": "openhim1"
+        }
+      },
+      "mapping": {
+        "resourceType": {
+          "expr": "'Patient'"
+        },
+        "sex": "doc.sex",
+        "name": {
+          "expr": "[ { \"use\":\"official\", \"family\": doc.name , \"given\": [ doc.short_name ] } ]",
+          "optional": false
+        },
+        "identifier.use": {
+          "expr": "'usual'"
+        },
+        "identifier.value": {
+          "expr": "doc._id"
+        },
+        "date_of_birth": "doc.date_of_birth",
+        "telecom": {
+          "expr": "[{ \"system\":\"phone\", \"value\": doc.phone, \"use\":\"mobile\" } ]",
+          "optional": true
+        },
+        "active": {
+          "expr": true,
+          "optional": false
+        }
+      }
+    },
+    "FHIR_practitioner": {
+      "relevant_to": "doc.type === 'person' && (doc.role == 'chw' || doc.role == 'nurse' )",
+      "destination": {
+        "base_url": "http://localhost:5001",
+        "path": "/fhir/Practitioner",
+        "auth": {
+          "type": "basic",
+          "username": "interop-client",
+          "password_key": "openhim1"
+        }
+      },
+      "mapping": {
+        "resourceType": {
+          "expr": "'Practitioner'"
+        },
+        "gender": "doc.sex",
+        "name": {
+          "expr": "[ { \"use\":\"official\", \"family\": doc.name , \"given\": [ doc.short_name ] } ]",
+          "optional": false
+        },
+        "identifier.use": {
+          "expr": "'usual'"
+        },
+        "identifier.value": {
+          "expr": "'cht/'.concat(doc._id)"
+        },
+        "birthDate": "doc.date_of_birth",
+        "telecom": {
+          "expr": "[{ \"system\":\"phone\", \"value\": doc.phone, \"use\":\"mobile\" } ]",
+          "optional": true
+        },
+        "active": {
+          "expr": true,
+          "optional": false
+        }
+      }
+    },
+    "FHIR_encounter": {
+      "relevant_to": "doc.type === 'data_record' && doc.form === 'interop_follow_up'",
+      "destination": {
+        "base_url": "http://localhost:5001",
+        "path": "/fhir/Encounter",
```

Comment:
We are updating FHIR directly without passing through the mediator. Do we want to allow CHT communicate directly FHIR database?